akka-stream-testkit/src/main/scala/akka/stream/testkit/StreamTestKit.scala

```diff
@@ -309,6 +317,9 @@ object TestSubscriber {
       self
     }
 
+    def expectMsgPF[T](f: PartialFunction[AnyRef, T]): T =
+      f(probe.expectMsgType[AnyRef])
```

Comment:
Yeah, make sense. Let me add tests and have two methods like expectEventPF and expectNextPF